tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -147,21 +149,21 @@ def print_result(result_prefix, return_code):
         if sys.stdout.isatty():
             suffix = "\033[31;1mFAILED\033[0m"
         else:
-            suffix ="FAILED"
+            suffix = "FAILED"
 
     print(result_prefix + suffix)
 
+
 def run_test(host, port, ssl_version, cipher, threshold):
     cipher_name = cipher.openssl_name
-
     failed = 0
     tcpdump_filter = "dst port " + str(port)
     tcpdump_cmd = ["sudo", "tcpdump", "-l", "-i", "lo", "-n", "-B", "65535", tcpdump_filter]
     tcpdump = subprocess.Popen(tcpdump_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
     ret = try_dynamic_record(host, port, cipher_name, ssl_version, threshold)
     # wait for pipe ready
-    sleep(2)
+    sleep(1)
```

Comment:
We've had problems with tcpdump hanging for a second when it initially starts up. Has this sleep been tested across many runs?

Reply:
I initiated the whole test 100 times and didn't have a problem with tcpdump - I looked back to when this was changed to try and understand why, but I just put it down to a general attempt to stop the test failing (which it no longer did once iId sorted the threshold issues.)

I'm not opposed to changing it back to 2, perhaps more tests are required.

Reply:
changed back in new commit, just incase.